ci/slab.toml

```diff
@@ -82,3 +82,8 @@ check_run_name = "WASM Client AWS Benchmarks"
 workflow = "csprng_randomness_testing.yml"
 profile = "cpu-small"
 check_run_name = "CSPRNG randomness testing"
+
+[command.code_coverage]
+workflow = "code_coverage.yml"
+profile = "cpu-small"
```

Comment:
don't we want the CPU big route for more parallelism ?

Reply:
It looks like the coverage is not parallelized, on my laptop it take only one CPU to run the tests.

Reply:
`cpu-big` profile doesn't significantly improve execution duration.
That said we still goes from 45mins on GitHub VM  down to 25mins on `cpu-small`. A good improvement though.

Reply:
yes I was missing the fact the coverage was not parallelized :/